webpack.config.client.js

```diff
@@ -109,6 +110,22 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
           );
         },
       ),
+      /*
+       * The webpack.ContextReplacementPlugin allows us to load only
+       * the specific locales we require into our code. By using this
+       * plugin we can reference `moment/locale/en.js` in the files
+       * we need it, without importing it in every file or importing
+       * every locale.
+       */
+      new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en-gb/),
```

Comment:
Did you consider using the `MomentLocalesPlugin` module that Moment.js recommend? Might be easier/safer to maintain than a regex solution.
https://momentjs.com/docs/#/use-it/webpack/

Reply:
Looking at that documentation, I agree that it's a better solution. I'll write up a separate issue to refactor in a future PR.